spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module AuthenticationHelper
+  def sign_in(user = nil)
```

Comment:
@malparty Signing in this way everytime can slow down the test, how about having this way of sign_in only of sign_in tests, and for remaining we just try to set the session. Refer to this project at nimble called mia, https://github.com/nimblehq/mia-web/blob/development/spec/support/devise_helpers.rb.

Reply:
Ok @ankitkalia1195 
What I've changed: 
- Review modules for the helper (more consistent)
- Rename the manual sign_in to "sign_in_ui" (which remain needed to test the login screens)
- Add a "sign_in" helper that utilize provided helpers
- Add a test (when user sign in and reach home page, it should not redirect him to sign in) in order to ensure the "sign_in" (non ui) helper works well

Reply:
@malparty Thanks, have you already committed these changes or yet to commit 🤔

Reply:
already committed/pushed ;-)